requirements.txt

```diff
@@ -1,3 +1,4 @@
 folium==0.2.1
 sphinx
 setuptools
+bokeh
```

Comment:
This was due to needing to dynamically import a dataframe for testing. I realize this probably isn't the best of ideas, but our test suite (I think it was Charts.ipynb) already uses this - it just wasn't imported before.

Reply:
I think the place we add requirements for testing is `tests_requires` in `setup.py`, rather than adding it to `requirements.txt`.  See if that looks right to you.

Reply:
You're right -  sorry about that!